bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java

```diff
@@ -241,6 +243,7 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                 var amendedHash = checkablePr.amendManualReviewers(localHash, censusInstance.namespace(), original);
                 addPrePushComment(pr, amendedHash, rebaseMessage.toString());
                 localRepo.push(amendedHash, pr.repository().url(), pr.targetRef());
+                processBackportLabel(pr, allComments);
```

Comment:
I think this call should be moved to the top of `markIntegratedAndClosed`. Otherwise, if the bot is interrupted just after line 245, then the next attempt at running `IntegrateCommand` will jump straight into `markIntegratedAndClosed` and skip the backport processing.

Reply:
I thought of this and call `processBackportLabel` before every `markIntegratedAndClosed` call, so I think the backport processing would not be skipped in any case. But put this method in `markIntegratedAndClosed` is still a good idea. I will do it.